nesta/production/orms/orm_utils.py

```diff
@@ -16,8 +16,7 @@
 import logging
 import time
 
-
-def insert_data(db_env, section, database, Base, _class, data):
+def insert_data(db_env, section, database, Base, _class, data, low_memory=False):
```

Comment:
can we raise an error from this function if the number of inserted/existing/duplicate rows does not equal the number of supplied rows? as discussed!